### ISSUE TITLE
it does work after i opened it

### ISSUE THREAD

**Issue by kitkatxd** [NQYub/Unturned-Bytex-Hack-v3#2]
idk what i did wrong but after opening it nothing happens

**Comment by ratdetector** [comment]
hey man you should factory reset your pc or if you want you can contact me on discord and i can help you remove the virus that you have install my user is guyonapc

**Comment by pythonsuc** [comment]
> hey man you should factory reset your pc or if you want you can contact me on discord and i can help you remove the virus that you have install my user is guyonapc

yes it a trojian i have a pc for this stuff you should get opara gx it will tell you what it is and its a trojian

**Comment by ratdetector** [comment]
> > hey man you should factory reset your pc or if you want you can contact me on discord and i can help you remove the virus that you have install my user is guyonapc
> 
> yes it a trojian i have a pc for this stuff you should get opara gx it will tell you what it is and its a trojian

hey just to let you know opera gx and anyother browser or anti virus sometimes gives false positives but high chance its gonna be a virus most of the time yes
